api/db/migrations/20230409194209_add_messageboard.ts

```diff
@@ -0,0 +1,68 @@
+import { Knex } from 'knex';
+
+const COLLATE = 'utf8mb4_unicode_ci';
+function applyCommon(table: Knex.CreateTableBuilder) {
+  table.collate(COLLATE);
+  table.increments('id').primary();
+  table.timestamps(false, true);
+}
+
+export async function up(knex: Knex): Promise<void> {
+  if (!await knex.schema.hasTable('messageboard')) {
+    await knex.schema.createTable('messageboard', table => {
+      console.log('Creating messageboard table');
+      applyCommon(table);
+            
+      table.integer('place_id', 10)
+        .unsigned()
+        .notNullable();
+      table.foreign('place_id')
+        .references('place.id');
+            
+      table.integer('member_id')
+        .unsigned()
+        .notNullable();
+      table.foreign('member_id')
+        .references('member.id');
+            
+      table.text('subject')
+        .notNullable();
+            
+      table.text('message')
+        .notNullable();
+            
+      table.integer('parent_id', 11)
```

Comment:
When trying to create a new message (not a reply), I receive an error that "parent_id" does not have a default value.

Reply:
I may have forgotten to update the migration to set a default null